community/cypher/cypher-compiler-2.3/src/main/scala/org/neo4j/cypher/internal/compiler/v2_3/InternalQueryStatistics.scala

```diff
@@ -28,8 +28,10 @@ case class InternalQueryStatistics(nodesCreated: Int = 0,
                            labelsRemoved: Int = 0,
                            indexesAdded: Int = 0,
                            indexesRemoved: Int = 0,
-                           constraintsAdded: Int = 0,
-                           constraintsRemoved: Int = 0) {
+                           uniqueConstraintsAdded: Int = 0,
+                           uniqueConstraintsRemoved: Int = 0,
+                           mandatoryConstraintsAdded: Int = 0,
+                           mandatoryConstraintsRemoved: Int = 0) {
```

Comment:
Should we keep track of separate constraints or is it enough to lump all constraints together in `InternalQueryStatistics`?